### ISSUE TITLE
Neural Network

### ISSUE THREAD

**Issue by williamjborom** [williamjborom/Combined-Cycle-Power-Plant-Predictor_Duke-AI-Product-Management#8]
Create a neural network with 2 hidden layers. Assess accuracy. Use SGD to train for 5 epochs. LR schedule = [0.01, 0.05, 0.1]

Layer 1 - Fully Connected 4x16
Layer 2 - Normalization Layer
Layer 3 - ReLU
Layer 4 - Fully Connected 16x16
Layer 5 - Normalization Layer
Layer 6 - ReLU
Layer 7 - Fully Connected 16x1
